homeassistant/components/sense/__init__.py

```diff
@@ -39,34 +43,90 @@
 )
 
 
-async def async_setup(hass, config):
-    """Set up the Sense sensor."""
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Sense component."""
+    hass.data.setdefault(DOMAIN, {})
+    conf = config.get(DOMAIN)
+    if not conf:
+        return True
+
+    hass.async_create_task(
+        hass.config_entries.flow.async_init(
+            DOMAIN,
+            context={"source": SOURCE_IMPORT},
+            data={
+                CONF_EMAIL: conf.get(CONF_EMAIL),
+                CONF_PASSWORD: conf.get(CONF_PASSWORD),
+                CONF_TIMEOUT: conf.get(CONF_TIMEOUT),
+            },
+        )
+    )
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up Sense from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
```

Comment:
This is not needed, already done in `async_setup`

Reply:
removed